cumulusci/cli/tests/test_cci.py

```diff
@@ -715,6 +715,18 @@ def test_project_dependencies(self):
 
         self.assertEqual("test:", "".join(out))
 
+    @mock.patch("cumulusci.cli.cci.doc_task")
+    def test_project_doc(self, doc_task):
+        runtime = mock.Mock()
+        runtime.universal_config.tasks = {"test": {}}
+        runtime.project_config.config = {
+            "project": {"name": "Test"},
+            "tasks": {"option": {"a": "b"}},
+        }
+        runtime.project_config.config_project = {"tasks": {"option": {"a": "b"}}}
+        run_click_command(cci.project_doc, runtime=runtime)
+        doc_task.assert_called()
```

Comment:
I think it would be a good idea to also assert that we are receiving the right output to `click.echo` for a given project configuration.

Reply:
Isn't this covered by `test_project_info` , especially since this doesn't create any output. Maybe it should, I'm open to change.

Reply:
Ya, that's fine with me.